packages/terra-application-navigation/src/ApplicationNavigation.jsx

```diff
@@ -417,24 +428,19 @@ const ApplicationNavigation = ({
         ref={contentLayoutRef}
         className={cx('content-layout', { 'drawer-menu-is-open': drawerMenuIsOpen })}
       >
-        {shouldRenderCompactNavigation(activeBreakpoint) ? renderCompactHeader() : renderHeader()}
-        <main
-          ref={mainContainerRef}
-          tabIndex="-1"
-          role="main"
-          className={cx('main-container')}
-          aria-labelledby={hiddenMainTitle ? 'main-inner-title' : null}
-        >
-          {hiddenMainTitle}
-          {children}
-        </main>
-        <Overlay
-          isOpen={drawerMenuIsOpen}
-          isRelativeToContainer
-          backgroundStyle="clear"
-          zIndex="6000"
-          onRequestClose={generateFocusToggle(handleRequestClose)}
-        />
+        <OverlayContainer className={cx('overlay-layout')} overlay={addOverlay()}>
```

Comment:
@lokesh-0813 Using the OverlayContainer here will start executing the `inert` logic that the Overlay uses to make the content behind the overlay inaccessible. This shouldn't be necessary, as we're using a FocusTrap for the DrawerMenu to ensure focus is trapped within the menu when open. I'm not sure if/how the addition of `inert` will change the overall accessibility of ApplicationNavigation as a whole, so I'm a little wary of the change.

I did notice that the Overlay element in this scenario was positioned offscreen. Notice the vertical offset in this screenshot:

![Screen Shot 2020-02-06 at 3 05 10 PM](https://user-images.githubusercontent.com/1711637/73978803-ad008780-48f2-11ea-8eb2-cb51f085d59b.png)

I'm wondering if just setting a `top: 0px` style on the Overlay will fix this?

Reply:
Hey @tbiethman , `top:0px` fixed the issue. I have updated here d1e0a651d4377e21498877b109fcda39d83854f1